src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -576,13 +595,13 @@ protected override void OnWorkspaceOpening(object obj)
             XmlDocument doc = obj as XmlDocument;
             if (doc != null)
             {
-                var camerasElements = doc.GetElementsByTagName("Cameras");
-                if (camerasElements.Count == 0)
+                var cameraElement = doc.GetElementsByTagName("Camera");
```

Comment:
I don't think you should do this - since we're no longer saving XML I wouldn't change XML deserialization - you must keep old graphs loading.